proto/sentry_protos/snuba/v1/endpoint_aggregate_bucket.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+enum Function {
+  FUNCTION_UNSPECIFIED = 0;
+  FUNCTION_SUM = 1;
+  FUNCTION_AVERAGE = 2;
+  FUNCTION_COUNT = 3;
+  FUNCTION_P50 = 4;
+  FUNCTION_P95 = 5;
+  FUNCTION_P99 = 6;
+  FUNCTION_AVG = 7;
+}
+
+message AggregateBucketRequest {
+  RequestMeta meta = 1;
+  TraceItemFilter filter = 2;
+  repeated AttributeAggregation aggregations = 3;
+  uint64 granularity_secs = 4;
+  repeated AttributeKey group_by = 5;
+}
+
+message AttributeAggregation {
+  Function aggregate = 1;
+  AttributeKey key = 2;
+  string label = 3;
+}
+
+message TimeSeries {
+  string label = 1;
+  // the names of the attribute keys which
+  // were in the group by
+  repeated string attribute_names = 2;
```

Comment:
Should this be `attribute_values`? Like we query for `p50()` grouped by `transaction` and `organization` this will tell us that for this timeseries its for the `/api` transaction in the `sentry` org?